### PR TITLE
26 feature info popup

### PR DESCRIPTION
**Pull request by Brian2397**
## Description

A HubCare Popup was created, following the colors according to the Visual Identity Document.
Popup shows a brief description of the 3 metrics and a Checkbox to disable or enable Hubcare.
For this, it was necessary to use the chrome.storage to store the checkbox variable that comes from the Popup.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Brian2397** [comment]
Does anyone have a look at the PR, please?
to see if everything is all right

File: app/scripts/contentscript.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ const repoName = window.location.pathname
 var content = saveClass("new-discussion-timeline experiment-repo-nav")
 var repoContent = saveClass("repository-content")
 let metrics = []
+var chaves = ""
```
Comment (cjjcastro) [inline app/scripts/contentscript.js]: Please keep the standardized code in English. Replace 'chaves' to 'keys' for example.

File: app/scripts/contentscript.js [hunk 2]
```diff
@@ -124,23 +131,28 @@ const requestMetrics = () => {
...
-    }, false);
 }
 
 const hubcarePage = () => {
     cleanPageContent()
     createCommitChart()
 }
-init()
+
+//init()
```
Comment (cjjcastro) [inline app/scripts/contentscript.js]: The hubcare is not starting, it seems that you commented the init

File: app/scripts/contentscript.js (hunk 1 above)
Comment (Brian2397) [reply app/scripts/contentscript.js]: Ok, the variable name no is 'popup_key'

File: app/scripts/contentscript.js (hunk 2 above)
Comment (Brian2397) [reply app/scripts/contentscript.js]: Was created new way to resolve this

**Comment by Brian2397** [comment]
All the requested changes, was resolved

File: app/scripts/contentscript.js (hunk 1 above)
Comment (cjjcastro) [reply app/scripts/contentscript.js]: seems solved

File: app/scripts/contentscript.js (hunk 2 above)
Comment (cjjcastro) [reply app/scripts/contentscript.js]: seems solved

**Pull request merged**